sha1/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+extern crate ewasm_api;
+extern crate sha1;
+
+#[cfg(not(test))]
+#[no_mangle]
+pub extern "C" fn main() {
```

Comment:
can you split the body of this function out into a helper, so that we could unit-test it without duplicating code?

Reply:
I'd rather not. Will unit test these when https://github.com/ewasm/ewasm-rust-api/issues/5 is finished.